target/linux/ramips/dts/mt7621_keenetic_kn-3013.dts

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "keenetic,kn-3013", "mediatek,mt7621-soc";
+	model = "Keenetic KN-3013";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &gmac0;
```

Comment:
It should refer to the MAC address from the device label. Usually this corresponds to the WAN port MAC address on Keenetic devices. In this case, it should be gmac1. Unfortunately, the devices that have already been added already contain this error, which needs to be fixed.